images/linux/scripts/base/apt.sh

```diff
@@ -20,6 +20,9 @@ echo "APT::Get::Assume-Yes \"true\";" > /etc/apt/apt.conf.d/90assumeyes
 # Uninstall unattended-upgrades
 apt-get remove unattended-upgrades
 
+#Update the cache
+apt-get update
```

Comment:
As I can see, `apt-get update` is already invoked in line 31.
It doesn't make sense to invoke it twice.

Let's put `add-apt-repository -y ppa:apt-fast/stable` before `apt update` and remove both 30 and 31 lines

Reply:
As Maxim mentioned above, please concat apt-get install into the one statement:
```
# Install aria2 and jq, apt-fast
apt-get install aria2 jq apt-fast
```